### ISSUE TITLE
Fix a couple of string ID conflicts

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#2911]
See:
```
Contradicting source strings for message with id 'vpn.connectionInfo.close'.
Contradicting source strings for message with id 'vpn.controller.info'.
```

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-1858)

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Andrea Marchesini What does QA need to check here? Thanks!

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

If all the strings are visible, we can consider this bug as fixed.

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Yes, something wrong is happening. Let me fix it in a different way.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

As we did not have details about what strings this issue is about, we have tracked this down separately: https://mozilla-hub.atlassian.net/browse/VPN-1916?searchSessionId=0c6e2a38-bac3-41e8-b327-7141fe67cf12&searchObjectId=264904&searchContainerId=10145&searchContentType=issue ( https://mozilla-hub.atlassian.net/browse/VPN-1916?searchSessionId=0c6e2a38-bac3-41e8-b327-7141fe67cf12&searchObjectId=264904&searchContainerId=10145&searchContentType=issue|smart-link ) ;

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

This issue should be fixed. Do you reproduce it on all the platforms? or just on windows?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

We reproduce this on latest VPN 2.8 release builds while using Windows, Mac OS and Linux.

As is a hovering issue, we cannot check it physically on mobile.

Thanks!

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified as fixed on the latest 2.8.0 (2.202203222138) VPN build, on Windows 10.

The CI strings are correct.

!image-20220323-083929.png|width=494,height=736!

!image-20220323-084108.png|width=425,height=258!

!image-20220323-084120.png|width=439,height=258!
